src/main/java/Task.java

```diff
@@ -0,0 +1,34 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
+        return this.description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+
+    public void markAsDone() {
+        isDone = true;
+    }
+
+    public void unmark() {
+        isDone = false;
+    }
```

Comment:
Perhaps markAsDone and unmark can be combined to be a setter for the isDone variable?